packages/@glimmer/debug/lib/render/logger.ts

```diff
@@ -53,12 +53,10 @@ export class DebugLogger {
   #lines(type: 'log' | 'debug' | 'group' | 'groupCollapsed', lines: FlushedLines): void {
     const [first, ...rest] = lines;
 
-    if (first) {
```

Comment:
The whole point of `FlushedLines` is to require at least one lint, so this check is unnecessary. ✂️